modules/mixins/loadingMixin.js

```diff
@@ -29,28 +29,30 @@
         base.call(this, args);
         this.loadedOnceDeferred = new $.Deferred();
         this.loadedOnce = false;
+        this.loadingCount = 0;
+        // Keeping loading boolean since it is technically public and may be in use.
```

Comment:
this is a comment that makes sense in the context of this patch but it kind of confusing later on. Maybe word it like:
"loading is a convenience flag that is the equivalent of loadingCount > 0"
You may also say it's deprecated if you'd like to remove it later, though I can see its use still

Reply:
see #341